app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class LikesController < ApplicationController
+  def create
+    like = Like.new(author: current_user, post_id: params[:post_id])
+    like.save
+    redirect_back(fallback_location: root_path) if like.save
```

Comment:
- [ ] After like save, it  is good to redirect to @post , rather than user_index